Game/Scripts/UITextTrigger.cpp

```diff
@@ -20,41 +30,79 @@ UITextTrigger::~UITextTrigger()
 
 void UITextTrigger::Start()
 {
-	player = App->GetModule<ModulePlayer>()->GetPlayer()->GetComponent<ComponentPlayer>();
 	componentRigidBody = owner->GetComponent<ComponentRigidBody>();
+	input = App->GetModule<ModuleInput>();
+
+	textBoxSize = textBox.size() - 1;
```

Comment:
```suggestion
	textBoxSize = static_cast<float>(textBox.size()) - 1.0f;
```

Avoids raising a warning.